index.css

```diff
@@ -27,6 +27,11 @@ article figure img {
   height: auto !important;
 }
 
+img[ix-sizes='auto'] {
+  display: block;
+  width: 100%;
+}
+
```

Comment:
same as [this comment](https://github.com/imgix/imgix.js/pull/297/files#r634844024)